app/presenters/account_show_presenter.rb

```diff
@@ -36,7 +36,8 @@ def show_password_reset_partial?
   end
 
   def show_manage_personal_key_partial?
-    user.encrypted_recovery_code_digest.present? &&
+    (user.encrypted_recovery_code_digest_multi_region.present? ||
```

Comment:
[suggestion] this conditional is repeated a few times, should it be a method on the user?

Reply:
Good call, I hadn't even realized that. Consolidated into a method on User in 1dcf6f1d911903896ddabebae9c6f479d90cd224